src/hooks/class-filters.php

```diff
@@ -216,6 +216,29 @@ interface Filters {
    */
   const MAILCHIMP = 'eightshift_forms/mailchimp';
 
+  /**
+   * Filter used for providing mailerlite credentials.
+   * IMPORTANT - Make sure to always return a string, even if $key isn't set.
+   *
+   * Example:
+   *
+   *   public function register(): void {
+   *     add_filter( 'eightshift_forms/mailerlite', [ $this, 'get_info' ], 1, 1 );
```

Comment:
```suggestion
   *     add_filter( 'eightshift_forms/mailerlite', [ $this, 'get_info' ], 1 );
```

The 1 can be omitted when only one argument is passed to the filter.